css/store.css

```diff
@@ -0,0 +1,238 @@
+/* store page */
```

Comment:
This is good that you've separated out the styles for this page - they'll be easier to manage